templates/partials/useful-links.html

```diff
@@ -0,0 +1,14 @@
+<div>
+        <h3>{{ useful_links_title }}</h3>
+
+        {% set link_text_array = ["Get started as an admin", "Deploy a node", "Marble Project Website", "Node Registry", "Python Client"] %}
```

Comment:
Isn't the marble project website this website??

Reply:
It's on the design right now.  I guess the final list will be decided later.